typed-racket-doc/typed-racket/scribblings/reference/types.scrbl

```diff
@@ -354,6 +354,12 @@ corresponding to @racket[trest], where @racket[bound]
 
 @ex[(box "hello world")]
 
+@defform[(Read-Only-Boxof t)]{A read-only @rtech{box} of @racket[t].}
+
```

Comment:
I think you want to elaborate a bit more here, especially explaining the relationship with regular `Box` types and subtyping. Maybe a discussion of the similarities with C's `const` keyword may help?